src/index.js

```diff
@@ -225,6 +217,22 @@ function getConfig(filePath, eslintPath) {
   }
 }
 
+async function resolvePrettierOptions(
+  filePath,
+  prettierPath,
+  prettierOptions = {},
+) {
+  /*
+   * The order of precedence (lowest to highest) is:
+   * - Options from a config file
+   * - prettierOptions
+   */
+
+  const prettier = getPrettier(prettierPath)
+  const optionsFromFile = await prettier.resolveConfig(filePath)
+  return Object.assign({}, optionsFromFile || {}, prettierOptions)
```

Comment:
Do we really want to extend the options here? Don't we wanna use the options from the file OR the prettier options from the settings view?